relay-metrics/src/view.rs

```diff
@@ -427,6 +424,46 @@ impl<'a> BucketView<'a> {
         Some(self)
     }
 
+    /// Estimates the number of bytes needed to serialize the bucket without value.
+    ///
+    /// Note that this does not match the exact size of the serialized payload. Instead, the size is
+    /// approximated through tags and a static overhead.
+    fn estimated_base_size(&self) -> usize {
+        50 + self.name().len() + aggregator::tags_cost(self.tags())
```

Comment:
Should we put this in a constant? For example

```suggestion
        BUCKET_OVERHEAD + self.name().len() + aggregator::tags_cost(self.tags())
```